libretro_core_options.h

```diff
@@ -892,12 +892,12 @@ struct retro_core_option_v2_definition option_defs_us[] = {
       BEETLE_OPT(widescreen_hack_aspect_ratio),
       "Widescreen Mode Hack Aspect Ratio",
       NULL,
-      "The aspect ratio that's used by the Widescreen Mode Hack.",
+      "Choose the aspect ratio to be used by the Widescreen Mode Hack.",
```

Comment:
Maybe "the aspect ratio to apply/force with the Widescreen Mode Hack"?

Reply:
"Choose the aspect ratio for the 'Widescreen Mode Hack'."?

Reply:
Much better and simpler.